src/haz3lschool/specs/Ex_OddlyRecursive_grading.ml

```diff
@@ -0,0 +1,2948 @@
+module SchoolExercise = GradePrelude.SchoolExercise
```

Comment:
remind me why we need a separate grading module? just because of prompt?

Reply:
I'm not sure whether there is a scalable way to grade students' editor states in browser, the IO sounds troublesome. But to run the grader in command line, we need the separation to remove web dependency.